scripts/1M_Glycerol_2nd_pulse.R

```diff
@@ -0,0 +1,25 @@
+###how to analyze only the second pulse when imaging two pulses (i.e. 1M Glycerol experiments) - creates csv file
+corrected_secondpulse <- function(genotype) {
+  #first get a directory path to use for saving:
+  message("choose your csv data file")  
+  filename <- file.choose()
+  
+  data <- read_csv(filename)
+  
+  pulse2 <- data %>%   
+    unnest() %>%   
+    filter(time > 90, time < 181)    #filter for data between 90 - 181 (second pulse)
```

Comment:
This part could be more generalizable, ie you could have an argument for something like pulse time, which could be the length of a single pulse, then use the length of the pulse to do the filter in line 11.